### ISSUE TITLE
Telegram Support

### ISSUE THREAD

**Issue by cron992** [Stealerium/Stealerium#139]
Discord detects malware such as Stealer and permanently bans the account, can you add Telegram support?

Regards.

**Comment by heckerthecat** [comment]
You really got perma banned on discord for sending file?

**Comment by cron992** [comment]
> Dosya göndermek için anlaşmazlıkta perma gerçekten yasaklandı mı?

Yes, my 2 accounts have been banned, discord detects this, Telegram support should be added
